server/db/driver/pg/upgrades.go

```diff
@@ -319,6 +324,39 @@ func v5Upgrade(tx *sql.Tx) (err error) {
 	return nil
 }
 
+// v6Upgrade creates the bonds table and its indexes on account_id and lockTime.
+func v6Upgrade(tx *sql.Tx) error {
+	bondsCreated, err := createTableStmt(tx, internal.CreateBondsTableV0, publicSchema, bondsTableName)
+	if err != nil {
+		return fmt.Errorf("failed to create bonds table: %w", err)
+	}
+	if bondsCreated {
+		log.Infof("Created new %q table", bondsTableName)
+	} else {
+		log.Warnf("Unexpected existing %q table!", bondsTableName)
+	}
+
+	namespacedBondsTable := publicSchema + "." + bondsTableName
+	err = createIndexStmt(tx, internal.CreateBondsAcctIndexV0, indexBondsOnAccountName, namespacedBondsTable)
+	if err != nil {
+		return fmt.Errorf("failed to index bonds table on account: %w", err)
+	}
+
+	err = createIndexStmt(tx, internal.CreateBondsLockTimeIndexV0, indexBondsOnLockTimeName, namespacedBondsTable)
```

Comment:
Why is there an index on locktime? Isn't it enough to just grab all the bonds for a certain user?

Reply:
`SelectActiveBondsForUser` uses `lockTime` in a condition to only choose bonds that haven't expired.

Another thought is to DELETE bonds occasionally to prune them when they become spendable (not just expired in the eyes of the DEX), but we still need to select bonds that are actually active even if they aren't refundable yet.